lib/reconcile/rabbitmq.go

```diff
@@ -220,15 +220,17 @@ func validateRabbitMQDefinition(rmq commontypes.AstarteRabbitMQSpec) error {
 	return nil
 }
 
-func getRabbitMQInitContainers() []v1.Container {
+func getRabbitMQInitContainers(dataVolumeName string) []v1.Container {
 	return []v1.Container{
 		{
-			Name:  "copy-rabbitmq-config",
+			Name:  "setup-rabbitmq",
 			Image: "busybox",
 			Command: []string{
 				"sh",
 				"-c",
-				"cp /configmap/* /etc/rabbitmq",
+				"cp /configmap/* /etc/rabbitmq ; " +
+					"cp /erlang-cookie-secret/.erlang.cookie /var/lib/rabbitmq/.erlang.cookie " +
```

Comment:
I  think also the first command should be followed by `&&` instead of `;`, otherwise if it fails, it would do so silently